confidential-component/attestation-service/README.md

```diff
@@ -0,0 +1,35 @@
+# Reproducible Build Attestation-service
+
+This project is based on Docker for reproducible build, and currently supports running on Ubuntu 22.04 platform. The attestation-service source code is located at [trustee/attestation-service at main · confidential-containers/trustee (github.com)](https://github.com/confidential-containers/trustee/tree/main/attestation-service). 
+
+## Files
+
+
+
+- `run.sh` main script. Use `./run.sh` to run.
+- `Dockerfile` to build docker and get binary.
+
+
+
+## Prerequisites
+
+
+
+- Ensure Docker is installed on your system
+- Install libtss2-dev and libsgx-dcap-quote-verify

```

Comment:
Since the build process is inside container, do we still need these packeages in host?